app/grants/models.py

```diff
@@ -385,13 +385,6 @@ def _do_helper_via_web3(self, fn, minutes_to_confirm_within=5):
         signed_txn = web3.eth.account.signTransaction(tx, private_key=settings.GRANTS_PRIVATE_KEY)
         return web3.eth.sendRawTransaction(signed_txn.rawTransaction).hex()
 
-    def do_cancel_subscription_via_web3(self, minutes_to_confirm_within=5):
```

Comment:
why was this removed?

Reply:
The miner should never handle a `cancelSubscription` call. This is a direct transaction on chain performed by the user.

Reply:
these changes are in the model, not in the miner..

Reply:
The rest of Grants calls the contract from javascript. `cancelSubscription` is handled in `grants/cancel_subsription.js` The miner is the only thing part of Grants that calls to the contract from the model. The miner will never call `CancelSubscription`. So it is unused code.

Reply:
while i agree that its not being called anywhere, i think its useful to have in the codebase in case we need it for administrative reasons.  checkout the precedent set at https://github.com/gitcoinco/web/pull/2725/files#diff-0552410df3354f05f06d86d522aab24fR622

Reply:
Makes sense